src/main/java/jenkins/plugins/git/traits/BranchDiscoveryTrait.java

```diff
@@ -79,6 +80,7 @@ public boolean includeCategory(@NonNull SCMHeadCategory category) {
     /**
      * Our descriptor.
      */
+    @Symbol("gitBranchDiscoveryTrait")
```

Comment:
No, symbols should not include redundant words or phrases which are already implied by the extension point being implemented. In this case that point is I suppose “trait” or “SCM source trait” so I would expect simply `gitBranchDiscovery`.

Reply:
I can update it, the name of the symbol does not matter to me.  As long as the root problem is fixed.